tests/external-links.js

```diff
@@ -364,28 +364,32 @@ async function updateGithubIssue(urlResults) {
    */
   function getBodyFromLinkData(linkData) {
     const scriptName = import.meta.url.split(`/`).slice(-2).join(`/`);
+    const rows = Object.entries(linkData).map(([url, statuses]) => {
+      const statusIcons = statuses.map(status => {
+        if (!status.failed) {
+          return `:heavy_check_mark:`;
+        }
+
+        const message = status.message.replaceAll(`\n`, ` `).replaceAll(`"`, `&quot;`);
+        const emoji = getFailedEmoji(status.message);
+        return `<a href="${status.jobUrl}" title="${message}">${emoji}</a>`;
+      }).join(`&nbsp;`);
```

Comment:
Could you extract this into a function please?

```js
/**
 * @param {LinkStatus} status The status to get the linked emoji for.
 * @returns {string} An emoji, wrapped in a link to the failed job if applicable.
 */
function getStatusEmojiLink(status) {
  if (!status.failed) {
    return `:heavy_check_mark:`;
  }

  const message = status.message.replaceAll(`\n`, ` `).replaceAll(`"`, `&quot;`);
  const emoji = getFailedEmoji(status.message);
  return `<a href="${status.jobUrl}" title="${message}">${emoji}</a>`;
}
```

```suggestion
      const statusIcons = statuses.map(status => getStatusEmojiLink(status)).join(`&nbsp;`);
```

Reply:
I can, which bit specifically? The `map` that runs to generate 'rows'?

Reply:
No, the `getStatusEmojiLink` function I commented above, which is currently an anonymous function inside `statuses.map`.

Reply:
Yes, no problem 👍